src/function_run_result.rs

```diff
@@ -1,15 +1,27 @@
 use colored::Colorize;
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 use std::{fmt, time::Duration};
 
+#[derive(Serialize, Deserialize)]
+pub struct InvalidOutput {
+    pub error: String,
+    pub output: String,
```

Comment:
such as `stdOutput`?

Reply:
I think that `stdout` should be considered as its own word rather than a short form that can be expanded into `stdOutput`. What about just `stdout`? Does that work?